build_tools/docker/bazel/Dockerfile

```diff
@@ -40,11 +41,11 @@ RUN apt-get install -y git
 # https://docs.bazel.build/versions/master/install-ubuntu.html
 ENV BAZEL_VERSION 2.1.0
 RUN apt-get install -y unzip zip wget \
-  && wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION?}/bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh \
-  && chmod +x bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh \
-  && ./bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh --user \
-  && rm bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh
-ENV PATH "/root/bin:$PATH"
+  && wget "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION?}/bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh" \
+  && chmod +x "bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh" \
+  && "./bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh" --user \
+  && rm "bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh"
+ENV PATH "/root/bin:${PATH}"
```

Comment:
I mean I guess it would be totally wild if PATH wasn't defined, but maybe add a `?` here as well :-D Basically I advocate for always writing variables as `"{FOO?}"` unless you have a good reason to write it differently. I think if PATH wasn't defined you'd end up with weirdness here with a trailing colon.

Reply:
I didn't even know `?` for error messages was a thing! Still can't believe that's not the default... (actually I can, it's bash).